app/src/DesignSystem/Button.svelte

```diff
@@ -2,12 +2,15 @@
   import Title from "./Title";
   export let variant = "primary";
   export let size = "big";
-  export let disabled = false;
+  export let disabled = null;
   export let icon = null;
+  export let style = null;
 
   let iconClass = icon ? (size === "big" ? "icon" : "icon-small") : null;
-  let disabledClass = disabled ? "disabled" : null;
-  let buttonClass = [variant, size, iconClass, disabledClass].join(" ");
+
+  // we want to dynamically change whether a button is disabled or not
+  $: disabledClass = disabled ? "disabled" : null;
```

Comment:
Why not have `disabled` be `null|"disabled"`.

Reply:
I don't think I follow. How would that work?